lib/dep-graph-builders/util.ts

```diff
@@ -0,0 +1,64 @@
+import { PackageJsonBase } from './types';
+import { DepGraphBuilder } from '@snyk/dep-graph';
+
+export interface PkgNode {
+  id: string;
+  name: string;
+  version: string;
+  dependencies: Record<string, { version: string; isDev: boolean }>;
+  isDev: boolean;
+}
+
+export const addPkgNodeToGraph = (
+  node: PkgNode,
+  isCyclic: boolean,
+  depGraphBuilder: DepGraphBuilder,
```

Comment:
Can we put the mutated element as first argument, keeps it clear in future if we add any more. Or return the depgraphbuilder to keep it explicit.

Reply:
good point !! also will change argument order for the other methods where `depGraphBuilder` is used ;)

Reply:
Thank you, just mutating args is a bit hard to see straight away so lets make it as clear as possible